test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/config"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type Manager struct {
+}
+
+func (m *Manager) GetConfig(ctx context.Context, c *kubernetes.Clientset, pod *corev1.Pod, containerName string) (*config.Config, error) {
+	var cmd []string
+	cfg := &config.Config{}
+	cmd = []string{
+		"/bin/cat",
+		config.CgroupMountPoint + "/cpuset/cpuset.cpus",
+		config.CgroupMountPoint + "/cpu/cpu.cfs_quota_us",
+		config.CgroupMountPoint + "/cpu/cpu.cfs_period_us",
+	}
```

Comment:
when crun is enabled , this creates a subdirectory called container which contains all the cpuset interface files,  Can we get those interface files too ?

Reply:
Yes. As discussed offline we'll have separate PR for it